.travis.yml

```
@@ -1,5 +1,9 @@
 language: swift
 osx_image: xcode10.2
 
+install:
+- brew install swiftlint
+
 script:
 - xcodebuild -workspace Example/Guppy.xcworkspace -scheme Example -sdk iphonesimulator build test
+- swiftlint
```

```suggestion
- swiftlint --strict
```

If we pass the strict flag, it will return an error code on any warnings or errors